player/lua.c

```diff
@@ -458,11 +456,6 @@ static int load_lua(struct mp_script_args *args)
 
     stats_register_thread_cputime(ctx->stats, "cpu");
 
-    if (LUA_VERSION_NUM != 501 && LUA_VERSION_NUM != 502) {
-        MP_FATAL(ctx, "Only Lua 5.1 and 5.2 are supported.\n");
```

Comment:
We know that other versions are not working, so why remove this check?

Reply:
Because it's a repetition of the check performed by Meson. It's fine to keep it if you prefer.

Reply:
It would be better to have run-time check for lua versions. Same as we do with ffmpeg

Reply:
`lua_version` was added in 5.2, so not much help...